### PR TITLE
New release v0.0.8

### PR DESCRIPTION
**Pull request by holgerd77**

### CODE REVIEW

**Comment by holgerd77** [comment]
Have also added the common branch protection rules to this repository.

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+(modification: no type change headlines) and this project adheres to
+[Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+
+## [0.0.8] - 2020-05-27
+
+This is a maintenance release with dependency and CI updates together with 
+some coder modernization changes.
```
Comment (ryanio) [inline CHANGELOG.md:12]: did you mean “code” instead of “coder”?

Comment (holgerd77) [reply CHANGELOG.md:12]: Yes, but "coder" sounds cool as well. 😄

Comment (holgerd77) [reply CHANGELOG.md:12]: (will nevertheless merge, this is such a tiny change)

**Comment by holgerd77** [comment]
Released as [v0.0.8](https://github.com/ethereumjs/ethashjs/releases/tag/v0.0.8) on npm.

**Pull request merged**